tracks/elixir/exercises/beer-song/mentoring.md

```diff
@@ -0,0 +1,76 @@
+### Reasonable solutions
+
+```elixir
```

Comment:
If we compare this to the suggested solution from `99 bottles` (the book), this doesn't seem to be thé reasonable solution we're looking for?

Reply:
It's a reasonable Elixir solution that maximises readability.